templates/search/blocks/search_sort_and_view_options.html

```diff
@@ -11,7 +11,7 @@ <h2 class="search-sort-view__heading">
         {{ form.sort_by.errors }}
         {{ form.sort_by }}
 
-        {{ 'sort_by'|include_hidden_fields:form }}
+        {{ 'catalogue-sort-form-desktop'|include_hidden_fields:form }}
```

Comment:
I've mentioned this API already. Just highlighting here so that it isn't missed.

Reply:
Same comment in relation to `hidden_fields_for_date_filter` and `bound field methods`